src/index.html

```diff
@@ -4,14 +4,162 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action = "https://mate-academy-form-lesson.herokuapp.com/create-application " method = "post">
+      <fieldset class = "group">
+        <legend>Personal information</legend>
+        <label class = "form_field">Surname
+          <input
+            type="text"
+            class= "field"
+            name= "surname"
+            autocomplete="off"
+            required
+            minlength="1"
+          >
+        </label>
+      
+        <label class = "form_field">Name
+          <input
+            type="text"
+            class= "field"
+            name= "name"
+            autocomplete="off"
+            required minlength="1"
+          >
+        </label>
+      
+        <label class = "form_field">How old are You?
+          <input
+            type="number"
+            class= "field"
+            name= "age"
+            value= "12"
+            min="1"
+            max= "100"
+          >
+        </label>
+      
+        <label class = "form_field"> Full date of birth:
+          <input
+            type="date"
+            class= "field"
+            name= "date_of_bt"
+            required
+          >
+        </label>
+          
+        <label for = "agree" class = "form_field"> I accept the term of the agreement
+          <input
+            type="checkbox"
+            name= "agreement"
+            id= "accept"
+            checked
+          >
+        </label>
+          
+      </fieldset>
+      
+      <fieldset class = "group">
+        <legend>Registration</legend>
+        <label class = "form_field">E-mail
+          <input
+            type="email"
+            class= "field"
+            name= "email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+      
+        <label class = "form_field">Password
+          <input
+            type="password"
+            class= "field"
+            name= "password"
+          >
+        </label>
+      </fieldset>
+      
+      <fieldset class = "group">
+        <legend>An interesting fact about you?</legend>
+        <label class = "form_field">Do you love cats?
+          <label>
+            Yes
+            <input
+              type= "radio"
+              name= "radiobut"
+              class= "field"
+              value= "Yes" 
+            >
+          </label>
+          <label>
+            No
+            <input
+              type= "radio"
+              name= "radiobut"
+              class= "field"
+              value= "No"
+              checked
+            >
+          </label>
+        </label>
+          
+        <label class = "form_field">What is you favorite color?
```

Comment:
```suggestion
        <label class="form_field">What is you favorite color?
```
Не надо добавлять тут пробелы